projects.js

```diff
@@ -0,0 +1,102 @@
+const projects = [
+  {
+    title: 'Multi Post Stories',
+    languages: ['html', 'Bootstrap', 'Ruby'],
+    imageUrl: './assets/Img_Placeholder_1.png',
+    desc: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    body: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+    id: 1,
+  },
```

Comment:
- [ ] Let's include a link to the `see the project` and a link to `see project source` for each project here so that we can dynamically add it to the `see live` and `see project`  `href` attribute.

```suggestion
  {
    title: 'Multi Post Stories',
    languages: ['html', 'Bootstrap', 'Ruby'],
    imageUrl: './assets/Img_Placeholder_1.png',
    desc: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
    body: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
    id: 1,
    seeLiveUrl: 'https://franciscoponcedev.github.io/Portfolio/',
    seeSourceUrl: 'https://github.com/franciscoPonceDev/Portfolio'
  },
```